### ISSUE TITLE
Add Frigate previews functionality to the frigate card scrubbing mode.

### ISSUE THREAD

**Issue by danishru** [dermotduffy/frigate-hass-card#1544]
**Is your feature request related to a problem? Please describe.**
The scrubbing mode is slow, it doesn't work well, especially on high-resolution cameras. On the iPhone, in the Home Assistant Companion app, it's very problematic to use the Frigate Card, when using the smartphone, it starts to heat up, the app freezes. It's almost impossible to quickly look at and decide whether to look at a section of the recording.

**Describe the solution you'd like**
It would be a great opportunity for the scrubbing mode to use the frigate previews functionality. There is even an http api for this https://docs.frigate.video/integrations/api#previews

**Comment by deliverer33** [comment]
I just wanted to validate this is what I'm experiencing currently. Live views I have zero issues, but if I play a recorded session back, it takes 30-60 seconds to load and sometimes doesn't load at all.  I don't have an issues in the frigate app directly and when I posted in Frigate help, it was suggested it was something on HA side causing an issue.  Please validate this open issue is in reference to what I'm seeing.

**Comment by dermotduffy** [comment]
> but if I play a recorded session back, it takes 30-60 seconds to load and sometimes doesn't load at all. 

Maybe? If your media isn't loading at all, or taking 30-60 seconds to load, there's definitely something wrong, but it's hard to tell if previews will help. It quite possibly might since they're much smaller and lightweight for your device to scan through especially for low-power devices.  Do you have the same problems using the HA media browser ("Media" tab on the left sidebar). A good test might be to find a recording that doesn't work in the Frigate card, then find it in the HA media browser. If it doesn't load in either, then your issue is somewhere between the integration and the Frigate server. If it does work in HA Media browser, but doesn't work in the card, then it's a card problem.

Bottom line: Recordings, as-is, should load. If they don't, that's a problem, and you should open a (different) issue for us to debug.

**Comment by deliverer33** [comment]
New to Frigate in HA.  Where is the media tab?  Is there a pic somewhere I can reference?  I’ll try that and then open a new issue if necessary.

**Comment by dermotduffy** [comment]
On the left-hand sidebar on the HA user-interface:

![image](https://github.com/user-attachments/assets/70b411b8-f8b5-4277-ab88-449bf1079f17)

**Comment by deliverer33** [comment]
Ahh thank you.  Unfortunately no improvement.  I’ll open a new thread on it.
